assembl/tests/fixtures/base.py

```diff
@@ -207,17 +207,49 @@ def test_app(request, admin_user, test_app_no_perm):
 
 
 @pytest.fixture(scope="function")
-def test_app_no_login(request, admin_user, test_app_no_perm):
+def test_app_no_login(request, test_app_no_perm):
     """A configured Assembl fixture with permissions
     and no user logged in"""
     config = testing.setUp(
         registry=test_app_no_perm.app.registry,
         settings=get_config(),
     )
+
     dummy_policy = config.testing_securitypolicy(
-        userid=None, permissive=False)
+    userid=None, permissive=False)
     config.set_authorization_policy(dummy_policy)
     config.set_authentication_policy(dummy_policy)
+
+    return test_app_no_perm
+
+@pytest.fixture(scope="function")
+def test_app_no_login_real_policy(request, test_app_no_perm):
+    """A configured Assembl fixture with permissions
+    and no user logged in"""
+    config = testing.setUp(
+        registry=test_app_no_perm.app.registry,
+        settings=get_config(),
+    )
+
+    from ...auth.util import authentication_callback
+    from pyramid.authorization import ACLAuthorizationPolicy
+    from pyramid.path import DottedNameResolver
+    resolver = DottedNameResolver(__package__)
```

Comment:
Can't this be just `DottedNameResolver()` since it is not using any methods in this `__package__`? You're resolving on the named resolver below...